wsagent/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitExceptionMapper.java

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2015 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.git;
+
+import org.eclipse.che.api.git.exception.*;
+import org.eclipse.che.api.git.shared.ConflictExceptionError;
+import org.eclipse.che.dto.server.DtoFactory;
+import javax.inject.Singleton;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+
+import static org.eclipse.che.dto.server.DtoFactory.newDto;
+
+/**
+ * Exception mapper for all GIT exceptions
+ *
+ * @author Yossi Balan ( yossi.balan@sap.com)
+ */
+@Provider
+@Singleton
+public class GitExceptionMapper implements ExceptionMapper<GitException> {
```

Comment:
Format this class according to https://github.com/codenvy/codenvy-dev-resources/wiki/Formatting#for-dot--separator-break-always-comes-before-the-symbol